crates/iceberg/src/arrow/schema.rs

```diff
@@ -45,6 +45,11 @@ use crate::{Error, ErrorKind};
 /// When iceberg map type convert to Arrow map type, the default map field name is "key_value".
 pub(crate) const DEFAULT_MAP_FIELD_NAME: &str = "key_value";
 
+/// UTC time zone for Arrow timestamp type.
+pub const UTC_TIME_ZONE: &str = "+00:00";
+/// When iceberg map type convert to Arrow map type, the default map field name is "key_value".
+pub const MAP_KEY_VALUE_FIELD_NAME: &str = "key_value";
```

Comment:
how does this differ from DEFAULT_MAP_FIELD_NAME above? The descriptions in their comments are the same.